SimPPS/Configuration/python/PPSTransport_cff.py

```diff
@@ -0,0 +1,20 @@
+import FWCore.ParameterSet.Config as cms
+from SimG4Core.Application.g4SimHits_cfi import *
+g4SimHits.Generator.MinEtaCut = cms.double(-13.0)
+g4SimHits.Generator.MaxEtaCut = cms.double( 13.0)
+g4SimHits.Generator.HepMCProductLabel   = 'LHCTransport'
+g4SimHits.SteppingAction.MaxTrackTime = cms.double(2000.0)
+g4SimHits.StackingAction.MaxTrackTime = cms.double(2000.0)
+
+from IOMC.RandomEngine.IOMC_cff import *
+RandomNumberGeneratorService.LHCTransport.engineName   = cms.untracked.string('TRandom3')
+
+from SimTransport.PPSProtonTransport.TotemTransport_cfi import *
+
+from IOMC.EventVertexGenerators.VtxSmearedParameters_cfi import Realistic25ns13TeV2016CollisionVtxSmearingParameters 
+from IOMC.EventVertexGenerators.VtxSmearedParameters_cfi import Realistic25ns13TeVEarly2017CollisionVtxSmearingParameters
+from IOMC.EventVertexGenerators.VtxSmearedParameters_cfi import Realistic25ns13TeVEarly2018CollisionVtxSmearingParameters
+
+LHCTransport.VtxMeanX  = Realistic25ns13TeV2016CollisionVtxSmearingParameters.X0
+LHCTransport.VtxMeanY  = Realistic25ns13TeV2016CollisionVtxSmearingParameters.Y0
+LHCTransport.VtxMeanZ  = Realistic25ns13TeV2016CollisionVtxSmearingParameters.Z0
```

Comment:
What if a different simulated beamspot is used?

Reply:
There is a new proton transport in the queue which PR is to be submitted as soon as this one gets approved. It involved a lot of other changes, including the use of Eras in order to properly configure the proton transport, a redesign of the PPSProtonTransport module to accommodate a new method for the transport. I would kindly ask to keep the file above as it 
is right now to avoid a yet more complicated situation.

Reply:
Ok. I just wanted to point out that this approach does not work in general, if it is being addressed in the mean time, that's great!

Reply:
Yes. To make it clearer, there is now 3 method for the proton propagation: the hector based, using ideal optics (basically for acceptance studies for future optics), the old totem parameterization, which uses real, validated optics (whichs is already in the release but works only for 2016 data) and the new one based also on validated optics, but using optical functions from database.